### PR TITLE
Split long insight into smaller ones (3)

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/python/functional-programming/arrays-ii/the-sorted-built-in-function.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/python/functional-programming/arrays-ii/the-sorted-built-in-function.md?ref=3bf4d9429e82df2bef787de3fda88107f990d269)

File: python/functional-programming/arrays-ii/the-sorted-built-in-function-ii.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+```python
+pyString = ',.()-.12345ABCDabcd='
+print(sorted(pyString))
+# result:
+# ['(', ')', ',', '-', '.', '.',
+#  '1', '2', '3', '4', '5', '=',
+#  'A', 'B', 'C', 'D',
+#  'a', 'b', 'c', 'd']
+```
```
Comment (kapnobatai137) [inline python/functional-programming/arrays-ii/the-sorted-built-in-function-ii.md]: this should've been in part 1. at the moment part 2 has ~320 words, so moving this section to part 1 will reduce that to a more acceptable value

Comment (kapnobatai137) [reply python/functional-programming/arrays-ii/the-sorted-built-in-function-ii.md]: should be inserted before this paragraph:
> Behind the scenes, Python calls a derivative of merge-sort on the list (Timsort[1]) ...

Comment (Stefan-Stojanovic) [reply python/functional-programming/arrays-ii/the-sorted-built-in-function-ii.md]: When you remove the code snippets, the first one has 214 and the 2nd 259.  I thought the code snippets don't count towards the total word count of an insight? I made the requested change, just wondering about the wordcount

File: python/functional-programming/arrays-ii/the-sorted-built-in-function-ii.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+
+---
+
+# The `sorted` Built-in Function (Part 2)
+
+---
+## Content
+
+Let's look at keys. Defining keys allows us to tell Python exactly what to compare to achieve the sort if we don't want Python to simply compare the two elements as they are. Below, we define a custom class of `Order`s, where each `Order` has a name and a number. We populate a list of orders and then define a function that returns the order number from a given order. We use this function as the key. This allows `sorted` to sort the list of orders by their number.
```
Comment (kapnobatai137) [inline python/functional-programming/arrays-ii/the-sorted-built-in-function-ii.md]:
```suggestion
Now, let's look at using keys together with the `sorted` method. Defining keys allows us to tell Python exactly what to compare to achieve the sort if we don't want Python to simply compare the two elements as they are. Below, we define a custom class of `Order`s, where each `Order` has a name and a number. We populate a list of orders and then define a function that returns the order number from a given order. We use this function as the key. This allows `sorted` to sort the list of orders by their number.
```

**Pull request merged**